public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap

```diff
@@ -1489,9 +1491,11 @@ exports[`Dashboard renders in flyout 1`] = `
                                                 <div
                                                   className="euiText euiText--medium"
                                                 >
-                                                  <EuiText>
+                                                  <EuiText
+                                                    size="s"
+                                                  >
                                                     <div
-                                                      className="euiText euiText--medium"
+                                                      className="euiText euiText--small"
```

Comment:
not you: but weird that this is basically the same style twice - could simply by removing a level if there's no risk, but up to you

Reply:
So one of the `EuiText` is from `EuiEmptyPromt`, and it comes default as size="m", this is not exposed as an adjustable prop of `EuiEmptyPromt`. Only title size is exposed. https://oui.opensearch.org/1.5/#/display/empty-prompt

So if we still want to have the small text without updating Oui, I think this might be the only way to go.

Reply:
Sounds good - also missed this was a snapshot file